lib/vue-simple-suggest.vue

```diff
@@ -37,13 +49,15 @@
             :query="text">
             <span>{{ displayProperty(suggestion) }}</span>
           </slot>
-        </div>
-
-        <slot name="misc-item-below"
-          :suggestions="suggestions"
-          :query="text"
-        ></slot>
-      </div>
+        </li>
+
+        <li>
```

Comment:
This <li> element needs to have a `v-if` to check for the slot below. This is needed to avoid empty <li> element in a final rendered component, if no misc-item slot is given.

Like this:
```html
<li v-if="$scopedSlots['misc-item-below'] && $scopedSlots['misc-item-below']()">
  ...
</li>
```

Same is true for the `misc-item-above` slot.